apps/src/studio/studio.js

```diff
@@ -801,7 +801,7 @@ Studio.onTick = function() {
 
   Studio.clearDebugElements();
 
-  var animationOnlyFrame = Studio.midExecutionFailure ||
+  var animationOnlyFrame = Studio.pauseInterpreter ||
```

Comment:
Would be good to use a name that is not specific to the interpreter, since we need this to impact both blockly/non-interpreter and droplet/interpreter modes. I *think* it probably already works for the blockly case, but the name is a little misleading.